### PR TITLE
Update base-pt-BR.yaml

### PR DESCRIPTION
**Pull request by HyronBit**
Fixed a few typos, and translated reward_storage and reward_miner_chainable

### CODE REVIEW

File: translations/base-pt-BR.yaml [hunk 1]
```diff
@@ -636,10 +636,10 @@ storyRewards:
...
     reward_miner_chainable:
         title: Extrator em Cadeia
-        desc: "You have unlocked the <strong>chained extractor</strong>! It can
-            <strong>forward its resources</strong> to other extractors so you
-            can more efficiently extract resources!<br><br> PS: The old
-            extractor has been replaced in your toolbar now!"
+        desc: Você desbloqueou o <strong>extrator em cadeia</strong>! Ele 
+            <strong>encaminha os recursos</strong> para outros extratores para que você
+            possa extrair recursos de forma mais eficiente!<br><br> OBS: O velho
+            extrator foi substituído na sua barra de ferramentas!"
```
Comment (ZrubekLukas) [inline translations/base-pt-BR.yaml:642]: Looks like you are missing the starting quotes, that's why you get one of the lint-check errors. Also on line 639, there is a trailing space at the end, that is the second lint-check error.

Comment (HyronBit) [reply translations/base-pt-BR.yaml:642]: Thanks, gonna fix them right now

Comment (HyronBit) [reply translations/base-pt-BR.yaml:642]: Actually can't figure out how to edit the file so I'm gonna do this in like 8 hours

Comment (ZrubekLukas) [reply translations/base-pt-BR.yaml:642]: When you click here in the PR on the files changed, there should be on right top of the file three dots "..." and under that option to edit the file.

Comment (HyronBit) [reply translations/base-pt-BR.yaml]: I edited it, and made a commit, hopefully I did it right, could you review it please? Thanks in advance

Comment (ZrubekLukas) [reply translations/base-pt-BR.yaml]: Looks good 👍🏼

**Comment by tobspr** [comment]
Ty!

**Pull request merged**